### DISCUSSION TITLE
I can't create a new project for a self-hosted Gitlab project

### DISCUSSION THREAD

**Discussion by jaequery** [vercel/community#4402] [Help] [answered]
### Summary

Help, I can't create a new project for a self-hosted Gitlab project.
When I click on Create New Project, it does not give me an option to by-pass Git integrations.
This is a private project, NOT public.
I am deploying via CLI so I do not need it to be connected.
How can I do this?

### Steps to Reproduce

- Click on create New Project
- You are now taken to Import Git Repository page
- I can not find any option to either connect a private self-hosted Gitlab repo or by pass it

**Comment by amyegan** [comment, marked as answer]
You should be able to create a new project using the CLI. If you run [`vercel`](https://vercel.com/docs/cli/project-linking#) from the repo on local, that should prompt you with the option to create a new project to deploy the repo. You should also be able to create a new project with the [`vercel project add`](https://vercel.com/docs/cli/project) command.

Please let me know if that works for you 🙏
